CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project uses [Semantic Versioning](http://semver.org/).
 
+# [1.3.0] - 2022-3-04
+### Added
+ - ``negative_sample_type`` to ``Interactions`` and ``DataLoader`` classes to have the abilty to negative sample users or items
+ - ``BasePipeline`` inheirits  ``negative_sample_type`` from the ``train_loader``
+ - ``negative_sample_type`` arguements in ``mapk``, ``mrr``, and ``auc`` and the calculations are computed accordingly
```

Comment:
Typo. 

```suggestion
 - ``negative_sample_type`` arguments in ``mapk``, ``mrr``, and ``auc`` and the calculations are computed accordingly
```

Reply:
I'm also not sure we need to add the `and the calculations are computed accordingly` there since I think that is implied.